### ISSUE TITLE
custom chromosomes with . in name

### ISSUE THREAD

**Issue by saycheeeeese** [biocore-ntnu/epic#54]
Hi, 

My chromosome names unfortunately contain full stops (chr1_v2.1, chr_v2.1 etc). When I use --chromsizes option and a file containing chromosome names and length I get an error.  When I remove the full stops this is resolved, but the chromosome names are incorrect now. 

 In the Genome.py script:

chromosome_lengths = [l.split() for l in open(chromsizes).readlines()]

appears not just to be splitting with white space, the full stops are causing a problem here for some reason. 
Any chance of a solution? 

Thanks!

**Comment by endrebak** [comment]
Please post the output of 

`cat -et chromsizes`

and the error message you get running the script.

**Comment by saycheeeeese** [comment]
Error is
`Traceback (most recent call last):
  File "/home/ubuntu/.local/bin/epic", line 216, in <module>
    args.chromosome_sizes = create_genome_size_dict_custom_genome(args.chromsizes)
  File "/usr/local/lib/python2.7/dist-packages/bioepic-0.1.25-py2.7.egg/epic/config/genomes.py", line 47, in create_genome_size_dict_custom_genome
    chromosome_lengths = [l.split() for l in open(chromsizes).readlines()]
IOError: [Errno 2] No such file or directory: 'Tbrucei_hybrid_chromsizes'
`
```
cat -et chromsizes
cat: chromsizes: No such file or directory
```

**Comment by endrebak** [comment]
According to the error message you emailed epic is not able to find your chromsizes file :)

Are you sure you called it `Tbrucei_hybrid_chromsizes`? Perhaps there should be a dot at the end of the hybrid, not an underscore? 

Thanks for trying epic and reporting this btw.

**Comment by saycheeeeese** [comment]
Ahaha I know sorry it was the wrong error message (I ran it again in the wrong directory by mistake), hence why I deleted the post.  I somehow fixed the problem however and am re running epic now.   
For some reason deleting the dots and then adding them back in and re-saving the file (done in sublime) produced a format epic could use.  If I ever work out why that made a difference I'll let you know!
